### ISSUE TITLE
We need a page for curriculum remixed by the IMLS pilot project

### ISSUE THREAD

**Issue by chadsansing** [mozilla/learning.mozilla.org#2433]
We need somewhere to share library remixes of our curriculum coming out of the IMLS project. Not all of the remixes are like our traditional modules, and it's important for pilot members and future contributors to find the work.

cc @anmechung

**Comment by anmechung** [comment]
+1 and adding @ibgmozilla

**Comment by hannahkane** [comment]
Can you say more about this?
 * Is there an existing page on the site that we could model this on?
 * Is the content ready? 
 * What's the timeline for this?

**Comment by chadsansing** [comment]
- It would look like /activities jr. Maybe also like this Thimble template: https://thimbleprojects.org/chadsansing/192408/
- @anmechung  & @ibgmozilla - best link for collected content? Discourse?
- Timeline is ASAP.

**Comment by jgmac1106** [comment]
Have we changed the ToS of Thimble to allow greater licensing choice?

Based on ToS all projects get a CC-BY-SA license. So people shouldn't remix
or share anything with a more restrictive license.

**Comment by chadsansing** [comment]
@jgmac1106, did you file this in the Thimble repo, as well? I think our scope here is pretty limited to making and hosting a page for the IMLS pilot members' work.

**Comment by jgmac1106** [comment]
Will do. It   just caught my attention when I saw the "Thimble Teaching Kit
Template. The ToS in Thimble mean ILMS participants should not be remixing
anything with a more restrictive license than CC-BY-SA and should know all
of their content is shared with a CC-BY-SA with Mozilla getting an
unlimited license to the work.

**Comment by hannahkane** [comment]
Would the page be accessible to anyone? Would we link to it from the Learning site? Would we be adding content to it frequently?

Where are we with the initial content? I think it will be easier to discuss if we can look at the content itself.

**Comment by chadsansing** [comment]
@hannahkane, yes the page would be public. We would likely link to it from the additional resources section of /activities, though @anmechung and @ibgmozilla might have another idea of how to approach this. I'm not sure how frequently we'd add to the page, but I imagine the upcoming Web Literacy Leaders program would produce content for it.

Remixes so far live here: https://docs.google.com/spreadsheets/d/1D4iRUsIwL9d0XbwFDJpNWUXCmmaeDFGM0kRyJCVVsM0/edit#gid=0.

**Comment by hannahkane** [comment]
Thanks, @chadsansing.

Can we clarify the purpose of posting these remixes? If the purpose is specific to the IMLS project—e.g. to showcase the work coming out of it, or to be a resource to participants—I think it makes sense to create a webpage specifically for these, as this ticket requests. If, however, the purpose is to provide useful materials to a broader audience, I think we should consider a different way of presenting these. For example, should the YouTube video that was offered as an additional resource for the Ping Kong activity be linked to from the Ping Kong activity itself? I would think it would reach more people there. Should some of the other materials be listed on the main activities page, perhaps with some brief text explaining the IMLS project, if that's important?

Can we get some help with the content? I think we'd need to provide some background info / context on the IMLS project in general, as well as each of the specific remixes. 

For each remix, I would think we should have:
 * Title of the remix and author (+ link to author's twitter account, if applicable)
 * Original project source and author
 * Purpose of the remix / what's different
 * Brief description / something that clarifies what the user will see if they click the link (e.g. clarify that they'll see a YouTube video that can be used in conjunction with the existing Ping Kong lesson)
 * Link to content
 * Ideally, each remix would have an associated image (which we have the rights to)

Additionally, we'll have to adjust the accessibility of the documents. I didn't have access to several of them. If we're going to put them on the site, they'll need to be made public. At least one of the links was broken.

Are all of these remixes going to fall under our CC-BY license, or will we need to specify a different license in any cases?

**Comment by chadsansing** [comment]
I think @anmechung is in the best position to answer most of those questions.

From my perspective, ideally, we'd work with the authors of remixes we want to include to help them put those remixes into our templates.

**Comment by IBGMozilla** [comment]
I agree with Chad, but also in our commitment to the IMLS project/grant,
our goal was to create activities, curriculum remixed for a library
audience that would be open and accessible to all so that is a major
deliverable.

**Comment by hannahkane** [comment]
Should we have a call to talk through the details?

**Comment by IBGMozilla** [comment]
Yes, can we find time tmrw/Friday?

**Comment by hannahkane** [comment]
Per meeting today:
 * @anmechung, @IBGMozilla, and @chadsansing will work on doc with complete content set (see [comment above](https://github.com/mozilla/learning.mozilla.org/issues/2433#issuecomment-288157572) for an idea of what that doc might look like)
 * once doc is ready, we'll build the page, and link to it from learning.mozilla.org/activities
 * that page, or individual remixes, can also be featured on mozillapulse.org
